spot-client/src/common/remote-control/remoteControlClient.js

```diff
@@ -97,6 +98,23 @@ export class RemoteControlClient extends BaseRemoteControlService {
             });
     }
 
+    /**
+     * Creates a P2P signaling connection that if successfully established will be used to send/receive remote control
+     * commands.
+     *
+     * @param {boolean} isServer - Whether to create a server or a client P2P signaling connection.
+     * @protected
+     * @returns {void}
+     */
+    _createP2PSignalingConnection(isServer) {
+        super._createP2PSignalingConnection(isServer);
+
+        !isServer
```

Comment:
When is remoteControlClient going to get isServer true?

Reply:
In theory never, but you never know :P Just trying to follow the api.

Reply:
It will look better when client and server create the type then this flag will be gone